baybe/recommenders/meta/base.py

```diff
@@ -8,23 +8,24 @@
 from attrs import define, field
 
 from baybe.exceptions import DeprecationError
-from baybe.recommenders.base import Recommender, RecommenderProtocol
+from baybe.recommenders.base import RecommenderProtocol
 from baybe.recommenders.deprecation import structure_recommender_protocol
+from baybe.recommenders.pure.base import PureRecommender
 from baybe.searchspace import SearchSpace
 from baybe.serialization import SerialMixin, converter, unstructure_base
 
 
 @define
-class Strategy(SerialMixin, RecommenderProtocol, ABC):
-    """Abstract base class for all BayBE strategies."""
+class MetaRecommender(SerialMixin, RecommenderProtocol, ABC):
+    """Abstract base class for all meta recommenders."""
```

Comment:
Putting this here since I do not know a better place: We should have some place where we briefly explain the difference between Meta and Pure Recommenders. Could we maybe already have something here? Maybe in the class docstring, maybe in the module description(s)?
It's obvious that there will be additional explanation about this in the user guides at some point, but I'd appreciate if we already include a little bit right now.

Reply:
it will be 1) in the user guide and 2) at the top of the modules (base.py) where I've just added it

Reply:
added to __init__